.ps-rule/na-suppress.Rule.yaml

```diff
@@ -0,0 +1,31 @@
+---
+# Synopsis: Suppress Rules for Not Available resources
```

Comment:
Please list the new suppression in the docs, specifically this file https://github.com/Azure/ResourceModules/blob/main/docs/wiki/The%20CI%20environment%20-%20Pipeline%20design.md#psrule-pre-flight-validation-pipeline
In the PSRule section as a new bullet point in the suppression groups list

Reply:
done.